Makefile

```diff
@@ -40,6 +40,11 @@ endif
 
 EXTRA_DOCKER_ARGS+=-e GOPRIVATE='github.com/tigera/*'
 
+ifeq ("$(LOCAL_USER_ID)", "0")
+# The build needs to run as root.
+EXTRA_DOCKER_ARGS+=-e RUN_AS_ROOT='true'
+endif
+
```

Comment:
Can this be moved to Makefile.common?

Reply:
yeah, that's probably a better place.

Reply:
https://github.com/projectcalico/go-build/pull/122